.github/workflows/build.yml

```diff
@@ -412,14 +412,14 @@ jobs:
         path: ${{ github.workspace }}/.idf_tools
         key: ${{ runner.os }}-idf-tools-${{ hashFiles('.git/modules/ports/espressif/esp-idf/HEAD') }}-${{ steps.py3.outputs.python-path }}-20220404
     - name: Clone IDF submodules
-      run: |
-        (cd $IDF_PATH && git submodule update --init)
+      run: git submodule update --init $IDF_PATH
       env:
         IDF_PATH: ${{ github.workspace }}/ports/espressif/esp-idf
     - name: Install IDF tools
       run: |
-        $IDF_PATH/tools/idf_tools.py --non-interactive install required
-        $IDF_PATH/tools/idf_tools.py --non-interactive install cmake
```

Comment:
From what I read, "install required" is equivalent to bare "install".  so the effect of this is to skip installing cmake.  This has the effect of changing the version of cmake used. Before this, the version from esp-idf is used.  after this, a version installed by github is used.  Is it important to use a matching version of cmake, or will any version do?

Reply:
I replicated the `install.sh` script here which doesn't seem to install `cmake` but I think it is better to have the idf specific `cmake` so I'll revert this. Thanks for pointing it out.